scripts/run-integration-tests-npm.sh

```diff
@@ -191,11 +191,14 @@ function install_npm_packages() {
     if [ -n "${npm_package_version}" ]; then
       npm install @hoprnet/hoprd@${npm_package_version}
     else
-      npm install ${tmp}/hoprd-package.tgz
+      # Some Node installations seems to miss node-pre-gyp
+      npm install node-pre-gyp
+      npm install ${tmp}/hopr-utils-package.tgz
+      npm install ${tmp}/hopr-connect-package.tgz
+      npm install ${tmp}/hopr-core-package.tgz
       npm install ${tmp}/hopr-ethereum-package.tgz
       npm install ${tmp}/hopr-core-ethereum-package.tgz
-      npm install ${tmp}/hopr-core-package.tgz
-      npm install ${tmp}/hopr-utils-package.tgz
+      npm install ${tmp}/hoprd-package.tgz
```

Comment:
Did you verify that the correct versions are being installed that way? The previous order was meant such that installed dependencies are replaced by the next package.

Reply:
Wait, let me check

Reply:
Thanks. Verified that it is working and added a comment.